examples/eris/src/index.js

```diff
@@ -1,14 +1,32 @@
-const Eris = require('eris');
+const Eris = require("eris");
+const fs = require("fs");
 const bot = Eris(`Bot ${process.env.DISCORD_TOKEN}`);
+const prefix = process.env.PREFIX || "!";
+const commands = fs
+  .readdirSync(__dirname + "/commands")
+  .filter(file => file.endsWith(".js"));
 
-bot.on('ready', () => console.log(`Logged in as ${bot.user.username}#${bot.user.discriminator}`));
+bot.on("ready", () =>
+  console.log(`Logged in as ${bot.user.username}#${bot.user.discriminator}`),
+);
 
-bot.on('messageCreate', (message) => {
-	if (message.content === '!ping') {
-		bot.createMessage(message.channel.id, 'Pong!');
-	} else if (message.content === 'Hello') {
-		bot.createMessage(message.channel.id, 'Choo choo! 🚅');
-	}
+bot.on("messageCreate", async message => {
+  if (!message.content.startsWith(prefix)) return;
+  if (message.author.bot) return;
+  const args = message.content.slice(prefix.length).split(/ +/);
+  const command = args.shift().toLowerCase();
+  if (!commands.includes(command + ".js")) return;
```

Comment:
```suggestion
  if (!commands.includes(`${command}.js`)) return;
```

Use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).